torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -918,7 +921,9 @@ def _register_post_forward_hooks(
         forward_handle.remove()
     state._post_forward_handles.clear()
     for module in modules:
-        module_param_handles = state._root_module_to_handles[module]
+        if module not in state._comm_module_to_handles:
+            continue
+        module_param_handles = state._comm_module_to_handles[module]
```

Comment:
nit: this can be `state._comm_module_to_handles.get(module, [])` to avoid one dict lookup

Reply:
same for the one above

Reply:
Let me do this in a follow-up to avoid re-running CI.